lib/lev/active_job.rb

```diff
@@ -2,6 +2,32 @@
   module Lev
     module ActiveJob
       class Base < ::ActiveJob::Base
+        ALWAYS_RETRYABLE = ->(exception) { true }
+
+        RETRYABLE_EXCEPTION_CONDITIONS = {
+          'ActiveRecord::RecordNotUnique' => ALWAYS_RETRYABLE,
+          'ActiveRecord::TransactionIsolationConflict' => ALWAYS_RETRYABLE,
+          'Delayed::WorkerTimeout' => ALWAYS_RETRYABLE,
+          'Faraday::TimeoutError' => ALWAYS_RETRYABLE,
+          'OpenSSL::SSL::SSLError' => ALWAYS_RETRYABLE,
+          'OpenStax::Exchange::ClientError' => ALWAYS_RETRYABLE,
+          'OpenStax::HTTPError' => ->(exception) {
+            status = exception.message.to_i
+            status == 0 || status >= 500
+          },
+          'OpenURI::HTTPError'  => ->(exception) {
+            status = exception.message.to_i
+            status == 0 || status >= 500
+          },
+          'OAuth2::Error'       => ->(exception) {
+            status = exception.response.status
+            status == 0 || status >= 500
+          },
+          'Redis::CannotConnectError' => ALWAYS_RETRYABLE,
+          'RuntimeError' => ALWAYS_RETRYABLE,
+          'SocketError' => ALWAYS_RETRYABLE
+        }
```

Comment:
Is there a way that individual jobs can add custom exceptions to retry (or not)? ditto at the Lev configuration level.  (possible that `OpenStax::Exchange::ClientError` would be something that Tutor would configure in its use of lev.